packages/foam-core/src/index.ts

```diff
@@ -9,16 +9,19 @@ import {
   isPlaceholder,
   isAttachment,
   getTitle,
+  NoteParser,
 } from './model/note';
 import { URI } from './common/uri';
 import { FoamConfig } from './config';
 import { IDataStore, FileDataStore } from './services/datastore';
 import { ILogger } from './utils/log';
 import { IDisposable, isDisposable } from './common/lifecycle';
 import { FoamWorkspace } from './model/workspace';
+import * as uris from './utils/uri';
```

Comment:
I find it difficult to parse the difference between this and URI on line 14. The change in casing perhaps is contributing to this.

A better name doesn't come to mind - maybe `uriHelper` based on the usage later? Unless the two can be merged? Curious to hear your thoughts. I'm not strongly worried about this one, but thought it worth mentioning.

Reply:
the goal of `uris` is to provide a namespace for all functions/types/.. related to uris.
`URI` is not part of that because it "existed" before (just like a couple of other uri related functions)..
but yeah, I should probably just make `URI` part of `uris`, I need to do a couple of refactoring PRs

does that address the question, or was there a wider point I am missing?

Reply:
Fine by me 👍🏻  Thank you for the explanation